src/frontend/types/libs/window.d.ts

```diff
@@ -1,11 +1,19 @@
+import { Participant } from '../Participant';
+
 // Ensure this is treated as a module.
 export {};
 
 declare global {
   interface Window {
     converse: {
+      acceptParticipantToMount: (participant: Participant) => void;
+      askParticipantToMount: () => void;
+      kickParticipant: (participant: Participant) => void;
+      rejectParticipantToMount: (participant: Participant) => void;
+      participantLeaves: () => void;
```

Comment:
Does it mean that any user which is able to access to `window.converse` could accept, kick, etc a participant?

Reply:
Yes anyine accessing `window.converse` can call these methods. Like it can send messages using `window.converse.env.$msg`

Reply:
But I can check is the current use is an `instructor`

Reply:
May be it could be done in a separate PR, but this sensitive method could check `roles` present in jwt?

Reply:
No I would do it in this PR